### PR TITLE
Standalone class to retrieve embedded images from document.

### PR DESCRIPTION
**Pull request by Jestemrique**
Added Standalone class "RetrieveEmbeddedImagesFromDocument".
Given SessionID, and documentID return ArrayList with images extracted from MD.

### CODE REVIEW

File: stand-alone/com/microstrategy/samples/beans/rwbean/DocumentExecution.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import com.microstrategy.web.beans.*;
 import com.microstrategy.web.objects.*;
 import com.microstrategy.web.objects.rw.*;
+import com.microstrategy.webapi.EnumDSSXMLObjectTypes;
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/beans/rwbean/DocumentExecution.java]: EnumDSSXMLObjectTypes doesn't seem to be needed in this class, guess it was accidentally added - can you remove it, just as a clean-up?

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	    //- Add 3rd parameter to getEmbeddedImagesFromDocument to filter by object type.
+	    //INclude object type with: EnumDSSXMLObjectTypes
+
+	}
+	
+	
+	 public static ArrayList<WebBlob> getEmbeddedImagesFromDocument(String documentID, WebIServerSession session) throws WebBeanException, WebObjectsException {
+		    //Usage: Saves the images embedded in the documentID to image file located in 'pathString'
+		  
+		    RWBean rwb = (RWBean)BeanFactory.getInstance().newBean("RWBean");
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Please fix the indent - in Eclipse for Mac, you can do it using cmd + I

OR via the menubar:
Screenshot 2019-07-31 at 14 31 01

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+			RWUnitDef[] rwUnitDefs = rwDefinition.findUnitsByType(EnumRWUnitTypes.RWUNIT_IMAGE);
+
+			//Prints number of images.
+			System.out.println("Number o fimages in document: " + rwUnitDefs.length);
+			
+			//Creating the ArrayList of WebBlog images objects.
+			RWImageDef imageBlobImpl = null;
+			String pathString = null;
+			WebObjectSource webObjectSource = null;
+			//WebBlob[] imagesBlobs = null;
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Please clean up & remove any unused comments, less noise in the sample 😄

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	    
+	    // The GUID of the document to use
+	    String documentID = "911D069848D92F6A07AF65A7658E97A0";
+	    
+	    // Create our I-Server Session
+	    WebIServerSession session = SessionManager.getSessionWithDetails(intelligenceServerName, projectName, microstrategyUsername, microstrategyPassword);
+
+	    //Array with the images objects.
+	    ArrayList<WebBlob> imagesArrayList = getEmbeddedImagesFromDocument(documentID, session);
+	    //TODO:
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Please clean up & remove any unused comments, less noise in the sample

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+	    String microstrategyPassword = "";
+	    
+	    // The GUID of the document to use
+	    String documentID = "911D069848D92F6A07AF65A7658E97A0";
+	    
+	    // Create our I-Server Session
+	    WebIServerSession session = SessionManager.getSessionWithDetails(intelligenceServerName, projectName, microstrategyUsername, microstrategyPassword);
+
+	    //Array with the images objects.
+	    ArrayList<WebBlob> imagesArrayList = getEmbeddedImagesFromDocument(documentID, session);
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Now that you have an ArrayList of WebBlob objects, what can you do with the WebBlob to save it to file? 

Would be good to add this to the demonstration, as not everyone will be familiar with WebBlob objects.

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java (hunk 2 above)
Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Indented.

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java (hunk 3 above)
Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Done.

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java (hunk 4 above)
Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Done.

File: stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java (hunk 5 above)
Comment (Jestemrique) [reply stand-alone/com/microstrategy/samples/beans/rwbean/RetrieveDocumentImages.java]: Created function to save the Array fo file system.

**Pull request merged**